spec/contracts.dd

```diff
@@ -7,9 +7,7 @@ $(HEADERNAV_TOC)
         $(P Contracts are a breakthrough technique to reduce the programming effort
         for large projects. Contracts are the concept of preconditions, postconditions,
         errors, and invariants.
-        Contracts can be done in C++ without modification to the language,
-        but the result is
-        clumsy and inconsistent.)
+	)
```

Comment:
```
Searching for trailing whitespace
Searching for tabs
./spec/contracts.dd:10:	)
```

There are a couple of tools that simply things a bit. For example, all D repos contain a `[.editorconfig`](https://github.com/dlang/dlang.org/blob/master/.editorconfig) file which specifies indent, tabs <-> whitespace etc.
Most IDEs/Editors support this automatically - even Emacs:
https://github.com/editorconfig/editorconfig-emacs